1. Пусть $X∼F(x)$ — случайная величина с произвольным распределением, неизвестным математическим ожиданием EX и известной дисперсией DX. Как выглядит доверительный интервал для EX с приближённым уровнем доверия 100(1−α)%?

A.         $\bar{X}_{n}\pm z_{1-\frac{\alpha}{2}}\frac{DX}{\sqrt{n}}$ </br>
</br>
B.         $\bar{X}_{n}\pm t_{n-1, 1-\frac{\alpha}{2}}\sqrt{\frac{DX}{n}}$ </br>
</br>
C.         $\bar{X}_{n}\pm z_{1-\frac{\alpha}{2}}{\sqrt\frac{DX}{n}}$ </br>
</br>
D.         $\bar{X}_{n}\pm z_{1-\alpha}\sqrt{\frac{DX}{n}}$ </br>

Պատասխան՝ C

2. Для 61 большого города в Англии и Уэльсе известны средняя годовая смертность на 100000 населения (по данным 1958–1964) и концентрация кальция в питьевой воде (в частях на миллион). Чем выше концентрация кальция, тем жёстче вода. Города дополнительно поделены на северные и южные.

In [ ]:
# Load water.txt file
import pandas as pd
import numpy as np
from scipy import stats

df = pd.read_csv("/content/water.txt", sep='\t')
df

location           town  mortality  hardness
0     South           Bath       1247       105
1     North     Birkenhead       1668        17
2     South     Birmingham       1466         5
3     North      Blackburn       1800        14
4     North      Blackpool       1609        18
..      ...            ...        ...       ...
56    South        Walsall       1527        60
57    South  West Bromwich       1627        53
58    South       West Ham       1486       122
59    South  Wolverhampton       1485        81
60    North           York       1378        71

[61 rows x 4 columns]

Постройте 95% доверительный интервал для средней годовой смертности в больших городах. Чему равна его нижняя граница? Округлите ответ до 4 знаков после десятичной точки. </br>
 </br>
  </br>
   </br>
**Будьте осторожны при использовании метода std()!** Дело в том, что у объекта numpy он по умолчанию вычисляется как
$\sqrt{\frac{1}{n}\sum_{i=1}^n(X_i-\bar{X})}$, а у объекта pandas — как $\sqrt{\frac{1}{n-1}\sum_{i=1}^n(X_i-\bar{X})}$. </br>
 </br>
Интересует только второй вариант, несмещённая оценка стандартного отклонения. </br>
 </br>
Чтобы не думать всё время о том, правильно ли вычисляется в вашем случае std(), можно всегда использовать std(ddof=1) (ddof — difference in degrees of freedom), тогда нормировка всегда будет на n-1.

In [ ]:
def conf_int(sample, st_dev):
    sample_size = len(sample)
    z_value = stats.norm.ppf(q=0.975)
    lower_bound = sample.mean() - z_value * (st_dev / np.sqrt(sample_size))
    upper_bound = sample.mean() + z_value * (st_dev / np.sqrt(sample_size))
    conf_int = (lower_bound, upper_bound)
    return conf_int

print(conf_int(df['mortality'], df['mortality'].std(ddof=1)))


(1477.0525038218827, 1571.2425781453305)


Ստորին սահմանը՝ 1477.0525

3. На данных из предыдущего вопроса постройте 95% доверительный интервал для средней годовой смертности по всем южным городам. Чему равна его верхняя граница? Округлите ответ до 4 знаков после десятичной точки.

In [ ]:
new_df = df[df["location"] == 'South']
conf_int(new_df["mortality"], new_df["mortality"].std(ddof=1))

(1322.8909479237761, 1430.7244366916086)

Վերին սահմանը՝ 1430.7244

4. На тех же данных постройте 95% доверительный интервал для средней годовой смертности по всем северным городам. Пересекается ли этот интервал с предыдущим? Как вы думаете, какой из этого можно сделать вывод? 

In [ ]:
new_df2 = df[df["location"] == 'North']
conf_int(new_df2["mortality"], new_df2["mortality"].std(ddof=1))

(1588.2335708894593, 1678.9664291105405)

Միջակայքերը չեն հատվում։ Հարավային շրջանում մահացության միջին մակարդակը ավելի ցածր է, քան հյուսիսայինում։

5. Пересекаются ли 95% доверительные интервалы для средней жёсткости воды в северных и южных городах?

A. Пересекаются </br>
B. Не пересекаются 

In [ ]:
print("South: ", conf_int(new_df["hardness"], df[df["location"] == 'North']["hardness"].std(ddof=1)))
print("North: ", conf_int(new_df2["hardness"], df[df["location"] == 'South']["hardness"].std(ddof=1)))

South:  (59.723639374595415, 79.81482216386614)
North:  (17.02873302344137, 43.77126697655863)


B. Не пересекаются 

6. Вспомним формулу доверительного интервала для среднего нормально распределённой случайной величины с дисперсией $\sigma^2$: </br>
$\bar{X}_{n}\pm z_{1-\frac{\alpha}{2}}\frac{\sigma}{\sqrt{n}}$ </br>
При $\sigma = 1$ какой нужен объём выборки, чтобы на уровне доверия 95% оценить среднее с точностью $\pm0.1$?

In [ ]:
import math
z_value = stats.norm.ppf(q=0.975)
math.ceil((z_value / 0.1)**2)

385

7. Объём выборки, необходимый для построения доверительного интервала заданной ширины: </br>
A. увеличивается с уменьшением требуемой ширины, </br>
B. уменьшается с ростом дисперсии выборки, </br>
C. увеличивается с увеличением требуемой ширины, </br>
D. увеличивается с ростом дисперсии выборки, </br>
E. увеличивается с ростом $\alpha$, </br>
F. уменьшается с ростом $\alpha$. </br>

Պատասխան՝ A, D, F

8. Давайте уточним правило трёх сигм. Утверждение: 99.7% вероятностной массы случайной величины $X∼N(\mu,\sigma^2)$ лежит в интервале $\mu±c⋅\sigma$. Чему равно точное значение константы c? Округлите ответ до четырёх знаков после десятичной точки.

In [ ]:
c = stats.norm.ppf(q=1-0.003/2)
c

2.9677379253417944

Պատասխան՝ 2.9677

9. Пусть $X∼N(\mu,\sigma^2)$. Какое распределение имеет величина $\frac{\bar{X}_n - \mu}{S_n/\sqrt{n}}$? </br>
A. $St(n-1)$ </br>
</br>
B. $N(0, 1)$ </br>
</br>
C. $\chi_{n-1}^2$ </br>

Պատասխան՝ A

10. Выберите все распределения с несимметричной функцией плотности: </br>

A. Фишера, </br>
B. Стьюдента, </br>
C. хи-квадрат, </br>
D. Гаусса

A. Фишера, </br>

C. хи-квадрат </br>


